cave/utils/convert_for_epm.py

```diff
@@ -1,6 +1,7 @@
 #!/bin/python3
 
 import numpy as np
+import traceback
```

Comment:
Is traceback still necessary in the file?

Reply:
No.